source/CommonJobs/CommonJobs.Application.EvalForm/DTOs/CalificationsEvaluationDto.cs

```diff
@@ -35,7 +35,9 @@ public class CalificationsEvaluationDto
 
         public List<string> Evaluators { get; set; }
 
-        public static CalificationsEvaluationDto Create(EmployeeEvaluation evaluation, List<string> evaluators, string currentPosition, string seniority)
+        public bool IsCompanyEvaluationDone { get; set; }
```

Comment:
I mean, _ReadyToDevolution_ status is when _Company evaluation_ is done but it is not still _OpenForDevolution_, right?